pkg/neorpc/result/raw_notary_pool.go

```diff
@@ -0,0 +1,56 @@
+package result
+
+import (
+	"encoding/json"
+
+	"github.com/nspcc-dev/neo-go/pkg/util"
+)
+
+// RawNotaryPool represents a result of `getrawnotarypool` RPC call.
+// The structure consist of `Hashes`. `Hashes` field is a map, where key is
+// the hash of the main transaction and value is a slice of related fallback
+// transaction hashes.
+type RawNotaryPool struct {
+	Hashes map[util.Uint256][]util.Uint256
+}
+
+// rawNotaryPoolAux is an auxiliary struct for RawNotaryPool JSON marshalling.
+type rawNotaryPoolAux struct {
+	Hashes map[string][]string `json:"hashes,omitempty"`
+}
+
+// MarshalJSON implements the json.Marshaler interface.
+func (p RawNotaryPool) MarshalJSON() ([]byte, error) {
+	var aux rawNotaryPoolAux
+	aux.Hashes = make(map[string][]string, len(p.Hashes))
+	for main, fallbacks := range p.Hashes {
+		mainString := main.StringLE()
```

Comment:
And BTW, it's not just `StringLE()`, use `MarshalJSON`, because you need the `0x` prefix.

Reply:
Firstly I missed this comment, sorry. MarshalJSON returns `[]byte`. So do you want to use `[]byte` as a key for map?

Reply:
Fixed.